guides/common/modules/proc_upgrading-packages-on-a-host.adoc

```diff
@@ -1,14 +1,28 @@
 [id="upgrading-packages-on-a-host_{context}"]
 = Upgrading Packages on a Host
 
-You can upgrade packages on a host in bulk in the {ProjectWebUI}.
+You can upgrade a single package or multiple packages on a host in the {ProjectWebUI}.
+The package upgrade process takes place via Katello agent or remote execution, depending on your configuration.
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
 . Click the name of the host you want to modify.
 . Click the *Content* tab, then click the *Packages* tab.
 . Select *Upgradable* from the *Status* list.
-. Select upgrade version from the dropdown menu in *Upgradable to* column where applicable.
+. Select an upgrade version from the dropdown menu in *Upgradable to* column where applicable.
 . Select the packages you want to upgrade.
-. Click the *Upgrade* button.
+. Click *Remote Execution* or *Customized Remote Execution* from the vertical ellipsis on the right of the package.
+** To upgrade multiple packages at a time, select the *Upgradable* filter from the *Status* box to list the upgradable packages.
```

Comment:
This line is duplicating information that is already in the steps 4. and 5.

Reply:
Addressed :+1: